CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+1. Use common sense
```

Comment:
I like this rule.

Reply:
I'll follow up with @lucasmbrown-usds to see if there is a standard code of conduct that we should use here.